libs/blocks/merch/merch.js

```diff
@@ -2,6 +2,7 @@ import {
   createTag, getConfig, loadArea, loadScript, loadStyle, localizeLink,
 } from '../../utils/utils.js';
 import { replaceKey } from '../../features/placeholders.js';
+import '../../deps/mas/commerce.js';
```

Comment:
Is this needed? This is also being imported in L427

Reply:
Yes, we want to start to load this ASAP when merch.js get loaded.
This is what task's description:
"Remove the related logic and move import of commerce.js to the top."